<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>leftpanel</class>
 <widget class="QWidget" name="leftpanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="interactive">
    <bool>false</bool>
   </property>
   <property name="transformationAnchor">
    <enum>QGraphicsView::AnchorViewCenter</enum>
   </property>
   <property name="resizeAnchor">
    <enum>QGraphicsView::NoAnchor</enum>
   </property>
   <property name="viewportUpdateMode">
    <enum>QGraphicsView::MinimalViewportUpdate</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>random</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>down</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>200</number>
   </property>
   <property name="autoRepeatInterval">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>up</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>200</number>
   </property>
   <property name="autoRepeatInterval">
    <number>100</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
